package.json

```
@@ -9,6 +9,7 @@
 		"build-release": "tsc -p src",
 		"prepublish": "safe-publish-latest && npm run build-release",
 		"test": "mocha",
+		"test-published-files": "rm -rf \"ecmarkup-$npm_package_version.tgz\" package && npm pack && tar zxvf \"ecmarkup-$npm_package_version.tgz\" && cp -r test package/test && cd package && mocha && cd .. && rm -rf \"ecmarkup-$npm_package_version.tgz\" package",
```

```suggestion
    "pretest-published-files": "rm -rf \"ecmarkup-$npm_package_version.tgz\" package",
		"test-published-files": "npm pack && tar zxvf \"ecmarkup-$npm_package_version.tgz\" && cp -r test package/test && cd package && npm test && cd ..",
		"posttest-published-files": "rm -rf \"ecmarkup-$npm_package_version.tgz\" package",
```
(using `npm test` instead of `mocha` ensures a single source of truth for the test command)

---

Will switch to `npm test`, but why the `pretest` change?

---

feel free not to do those; it just cleans up the script a bit, and makes it clearer which steps are "setup" and which are "teardown".

---

I'm fine with it, just wondered why.